tests/java/third_parties/elasticsearch/testdata/main.java

```diff
@@ -0,0 +1,59 @@
+// Use bearer:expected java_third_parties_elasticsearch to flag expected findings
+
+public class Foo {
+  public void bad(User user) {
+    // ...
+    ElasticsearchClient esClient = new ElasticsearchClient(transport);
+    // bearer:expected java_third_parties_elasticsearch
+    Customer customer = new Customer("cust-1", user.email, user.name);
+
+    esClient.indices().create(c -> c
+        .index("customers")
+    );
+    IndexResponse response = esClient.index(i -> i
+        .index("customers")
+        .id(customer.getSku())
+        .document(customer)
```

Comment:
I would have expected the finding to be L13 or L16. Seems a bit weird to show up L8.

Reply:
Agreed it's very weird. It doesn't correspond with what is shown in the report either. Since I am trying to complete these rules as quickly as possible I have not had time to investigate fully but my guess is that this is the data type as source, and the expected findings' location matches the source rather than the sink: 

```
{
  ...
  "findings": [
    {
      ...
      "id": "java_third_parties_elasticsearch",
      "title": "Leakage of sensitive data to ElasticSearch",
      "documentation_url": "https://docs.bearer.com/reference/rules/java_third_parties_elasticsearch",
      "line_number": 16,
      ...
      "source": {
        "start": 8,
        "end": 8,
        "column": {
          "start": 48,
          "end": 58
        }
      },
      "sink": {
        "start": 16,
        "end": 16,
        "column": {
          "start": 10,
          "end": 18
        },
        "content": "document"
      },
      "parent_line_number": 16,
      "snippet": "document",
      "fingerprint": "9e79e6ed90cb23ce3224f765e07c3417_0",
      "old_fingerprint": "9ecce1f90c722b28e0a9e8e6474e1789_0",
      "code_extract": "        .document(customer)",
      "severity": "high"
    }
  ],
  "expected_findings": [
    {
      "rule_id": "java_third_parties_elasticsearch",
      "location": {
        "start": 8,
        "end": 8,
        "column": {
          "start": 5,
          "end": 71
        }
      }
    }
  ]
}
```

Reply:
Ah yeah https://github.com/Bearer/bearer-rules/blob/main/tests/helper.js#L100
That should be `sink` instead

Reply:
I'll check that it does the right thing here https://github.com/Bearer/bearer-rules/pull/272 and we will be able to update this one

Reply:
Fixed!